Jenkinsfile

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env groovy
+
+pipeline {
+  agent none
+  environment {
+    BASE_DIR="src/github.com/elastic/beats"
+  }
+  options {
+    timeout(time: 1, unit: 'HOURS') 
```

Comment:
I think you are being a bit optimistic here :smile:

Reply:
Yup 🐼

Reply:
Sounds like a good target to not go above 😇 :-D